samples/sensor/fxos8700/prj.conf

```diff
@@ -7,3 +7,10 @@ CONFIG_SYS_LOG_SENSOR_LEVEL=4
 CONFIG_FXOS8700_MODE_HYBRID=y
 CONFIG_FXOS8700_TEMP=y
 CONFIG_FXOS8700_TRIGGER_OWN_THREAD=y
+
+CONFIG_FXOS8700_MODE_ACCEL=n
+CONFIG_FXOS8700_PM_LOW_POWER=n
+CONFIG_FXOS8700_DRDY_INT1=n
```

Comment:
This will break boards that use INT1

Reply:
Why? ``frdm_k64f`` and ``hexiwear_k64`` use default INT2. I have inserted it to make easier to test the sample on ``frdm_kw41z`` and ``reel_board``, CONFIG_FXOS8700_DRDY_INT1 has to be changed manually to ``y``. See also  test-accel section in samples/sensor/fxos8700/sample.yaml